arch/arm/boot/dts/wireless-murata-sd8987.dtsi

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2020 CompeGPS Team S.L.
+ * All rights reserved
+ *
+ */
+
+/ {
+	init-gpios {
```

Comment:
i'm not sure if we should be aware of non repeating group names (init-gpios) without the extended property (&). I know that it's a different situation when including inside "/ {"

init-gpios is also declared on parent dtsi (terra common, trailplus common)